src/printer.js

```diff
@@ -368,10 +368,14 @@ function printStatement(path, options, print) {
         return concat([
           "<?php",
           hardline,
-          path.call(childrenPath => {
-            return printLines(childrenPath, options, print);
-          }, "children"),
-          hardline
```

Comment:
Why not just do it with a one-line change?
```
...,
path.call(childrenPath => {
  return printLines(childrenPath, options, print);
}, "children"),
node.children.length ? hardline : ''
```

Reply:
@czosel avoid extra `call`, improve perf

Reply:
Hmm, that sounds like a microoptimization to me - the effect is (a) probably very small and (b) only present when you're formatting an empty php file. I'd prefer to optimize for readability.

Reply:
@czosel let's do this